pkg/config/setup/system_probe.go

```diff
@@ -280,6 +280,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Setup) {
 	cfg.BindEnv(join(smNS, "max_postgres_stats_buffered"))
 	cfg.BindEnvAndSetDefault(join(smNS, "max_postgres_telemetry_buffer"), 160)
 	cfg.BindEnv(join(smNS, "max_redis_stats_buffered"))
+	cfg.BindEnvAndSetDefault(join(smNS, "disable_map_preallocation"), true)
```

Comment:
This will enable by default the `BPF_F_NO_PREALLOC` for USM for Kafka. do we want this enablement by default (or maybe we want to start with selected clusters)? I remember @usamasaqib mentioning the need of adding some extra telemetry to be sure we are not hitting some performance degradation.

I do not want to slow this down, but it may be useful doing a gradual rollout of the change.

Reply:
We will have gradual rollout (nightly clusters first once this is merge, and then during 7.72 in about a month for about a month)

Reply:
it was deployed on oddish-c for the weekend, and I didn't notice any regression